content/docs/search/reactivesearch-api/reference.md

```diff
@@ -432,6 +432,30 @@ This property can be used to control (enable/disable) the synonyms behavior for
 | ------ | --------------------------- | -------- |
 | `bool` | `search`                    | false    |
 
+### rankFeature
+This property allows you to define the [Elasticsearch rank feature query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-rank-feature-query.html#query-dsl-rank-feature-query) to boost the relevance score of documents based on the `rank_feature` fields.
```

Comment:
@bietkul We should specify the shape of this object. Enumerate the three functions and what parameter values (and validations) are applied to each.

Add a second example that adds more than one functions.